src/main/groovy/com/brightsparklabs/gradle/docs/DocsPlugin.groovy

```diff
@@ -467,14 +473,20 @@ public class DocsPlugin implements Plugin<Project> {
                      * be overridden in Asciidoc documents. See:
                      * - https://docs.asciidoctor.org/asciidoc/latest/attributes/assignment-precedence/
                      */
-                    attributes \
-                            'chapter-label@'      : '',
-                            'icons@'              : 'font',
-                            'imagesdir@'          : project.file(config.buildImagesDir),
-                            'numbered@'           : '',
-                            'source-highlighter@' : 'coderay',
-                            'toc@'                : config.tocPosition,
-                            'title-logo-image@'   : config.titleLogoImage
+
+                    Map<String,Object> pluginAttributes = [
+                        "chapter-label@"        : '',
+                        "icons@"                : 'coderay',
+                        "numbered@"             : '',
+                        "source-highlighter@"   : 'coderay',
+                        "toc@"                  : config.tocPosition,
+                        "title-logo-image@"     : config.titleLogoImage,
+                        "imagesdir@"            : project.file(config.buildImagesDir)
```

Comment:
Revert the order to the original (sorted by keys). It's annoying having to search up and down to see what attributes are defined.

Also, use single quotes for every key. There's no reason to use double on one side and single on the other. We prefer single to make it explicit that string interpolation is not being performed.

Update the readme to match the alphabetical order.